### PR TITLE
binance: handleErrors: more invalid order cases & linting errors

### PR DESCRIPTION
**Pull request by mkutny**

### CODE REVIEW

File: js/binance.js [hunk 1]
```diff
@@ -795,6 +795,8 @@ module.exports = class binance extends Exchange {
         if (code >= 400) {
             if (code == 418)
                 throw new DDoSProtection (this.id + ' ' + code.toString () + ' ' + reason + ' ' + body);
+            if (body.indexOf ('Price * QTY is zero or less'))
```
Comment (kroitor) [inline js/binance.js]: It lacks >= 0 in this conditional... still have to have it, unfortunately.

Comment (mkutny) [reply js/binance.js]: Oh! Linter said nothing, will try to find the rule.

Comment (mkutny) [reply js/binance.js]: No luck with the linter (

Comment (kroitor) [reply js/binance.js]: No worries, I'll look into it, maybe we could make it transpile these conditionals without the `>= 0` clause. Thanks anyway for editing it!

Comment (mkutny) [reply js/binance.js]: > maybe we could make it transpile these conditionals without the `>= 0` clause

No, that's definitely my fault, we need `=` part of comparison anyway it's good that you spot it.

**Pull request merged**